index.rst

```diff
@@ -511,6 +511,25 @@ particular planning package. Thinking through the plan in this way can help in
 building up a detailed plan in a flexible, agile way, while also ensuring that
 scope, cost and schedule are carefully controlled.
 
+.. _sec-software-release:
+
+Software Releases
+-----------------
+
+Per :ref:`sec-cycle-cadence`, a series of software releases will be made
+throughout LSST construction. These will provide a stable basis upon which
+external users (other subsystems, science collaborations and the wider
+community) can base their work.
+
```

Comment:
I think we should come up with another term for a distribution of less-tested, less-characterized code.  I'd prefer to reserve "release", unqualified, for something we stand behind.  I imagine Chuck and Robert will likely be getting "hot-off-the-press" builds at a cadence much more frequently than even monthly.